probe-rs/src/probe/jlink/speed.rs

```diff
@@ -99,17 +99,16 @@ impl JLink {
     /// [`Capability::AdaptiveClocking`]. Note that adaptive clocking may not work for all target
     /// interfaces (eg. SWD).
     ///
-    /// When the selected target interface is switched (by calling [`JayLink::select_interface`], or
+    /// When the selected target interface is switched (by calling [`JLink::select_interface`], or
     /// any API method that automatically selects an interface), the communication speed is reset to
     /// some unspecified default value.
-    pub(crate) fn set_speed(&mut self, speed: SpeedConfig) -> Result<()> {
+    pub(super) fn set_jtag_speed(&mut self, speed: SpeedConfig) -> Result<()> {
```

Comment:
Why call this `jtag` when it also sets the `SWD` speed? :)

Reply:
Does it? 
![image](https://github.com/probe-rs/probe-rs/assets/977627/92d198f9-afa0-434e-9476-ea39432df410)

Reply:
I think this is just lazyness/outdated docs on their end. Before SWD rose to fame everything was just JTAG and that's also where the J-Link name comes from.
I think setting the speed just sets the clock speed regardless of protocol but I might be wrong here ofc. I doubt there is another API call to set the speed tho?

Reply:
Oki let's assume it sets SWD, too.